Looms.PoS/Looms.PoS.Application/Models/Requests/Product/Variation.cs

```diff
@@ -0,0 +1,4 @@
+namespace Looms.PoS.Application.Models.Requests.Product;
+
+//TODO: idk where to put this, so im having it here since both create and update use the same structure
+public record Variation(string Name, float? Price);
```

Comment:
Rename to `VariationRequest`

Reply:
fixed in "fixed indentation/layout problems, changed Variation to Variation request and made create and update requests take IEnumerable<VariationRequest>"